src/vaadin-grid.html

```diff
@@ -821,6 +852,7 @@
         }
       }
 
+      /** @private */
       __getRowModel(row) {
```

Comment:
This seems to be called from many files, should be `@protected`?

Reply:
Done.

Reply:
Should add `@return {!GridRowData}`

Reply:
Done.